conf/application.conf

```diff
@@ -72,11 +72,11 @@ metrics {
 }
 
 cache {
-  expiry = 120 # minutes
+  expiry = 15 # minutes
 }
 
 passcode {
-  expiry = 15 # minutes
+  expiry = 3 # minutes
```

Comment:
Praveen just said he would like 2 mins please

Reply:
i have updated it to 2 mins